hashring/hashring_test.go

```diff
@@ -217,8 +263,9 @@ func BenchmarkHashRingLookupN(b *testing.B) {
 
 	b.StartTimer()
 	for n := 0; n < b.N; n++ {
+		k := rand.Intn(100)
```

Comment:
randomness in general seems bad in benchmarks especially when we're talking orders of magnitude. why randomize?